assets/src/components/Label.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+function Label (props) {
+  const {
+    left,
+    color,
+    labelPlacement,
+    labelText
+  } = props
+
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        display: 'inline-block',
+        color,
+        left: labelPlacement === 'down'
+          ? `calc(${left} - 115px)`
+          : left,
+        zIndex: 1,
+        top: labelPlacement === 'down'
+          ? '55px'
+          : labelPlacement === 'downLower'
+            ? '75px'
+            : labelPlacement === 'up'
+              ? '-25px'
+              : 0
```

Comment:
Hacky but couldn't think of a better way

Reply:
What if you changed the `labelPlacement` to be a signed decimal? Then you could multiple by `25` and pass it to `'${pixels}px'`. It might also make the JSX more read-able if you moved the logic/calculation above the `return` and created a variable, but that's a style preference I'll leave up to you.